docs/en/5-templates-syntax.md

```diff
@@ -158,6 +181,10 @@ Both templates are applied. Result:
 
 `elemModVal` checked for compliance after converting to String. This behavior is similar to checking `modVal`.
 
+Second argument of `elemMods()` can be omited or be equal to '*'. In this case
```

Comment:
`elemMods` → `elemMod`

`mods` → `mod`

Personally I don't know why we should use `'*'` here as well as no argument. I don't see any reason for it. But I'm not against.